VideoLocker/build.gradle

```diff
@@ -180,6 +182,7 @@ android {
         // test configuration
         testApplicationId 'org.edx.mobile.test'
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "com.facebook.testing.screenshot.ScreenshotTestRunner"
```

Comment:
What does it mean to have two different ``testInstrumentationRunner``s? Will this break the end to end tests? Do we need to do something like this http://stackoverflow.com/a/31638726/4586460